ssl/statem/extensions.c

```diff
@@ -1533,7 +1533,7 @@ int tls_psk_do_binder(SSL *s, const EVP_MD *md, const unsigned char *msgstart,
         void *hdata;
 
         hdatalen = BIO_get_mem_data(s->s3->handshake_buffer, &hdata);
-        if (hdatalen <= 0) {
+        if (hdatalen == 0) {
```

Comment:
Why this change? BIO_ctrl can return negative values ...

Reply:
On line 1532, `hdatalen` is defined as `size_t`...  I'll think of a different solution.